script/Makefile

```diff
@@ -90,6 +91,15 @@ TEST_PREBUILD = build
 #GOLDFLAGS += -X github.com/apache/camel-k/pkg/util/olm.DefaultStartingCSV=
 #GOLDFLAGS += -X github.com/apache/camel-k/pkg/util/olm.DefaultGlobalNamespace=openshift-operators
 
+# Define target platform for docker build
```

Comment:
We better move this check inside the `images-arch` to make sure it will be exclusively used when it is called by that procedure.